vortex-expr/src/operators.rs

```diff
@@ -0,0 +1,128 @@
+use std::ops;
+
+use crate::expression_fns::binary_expr;
+use crate::expressions::Expr;
+
+#[derive(Copy, Clone, Debug, Eq, PartialEq, PartialOrd)]
+pub enum Operator {
+    // arithmetic
+    Plus,
```

Comment:
Why do we have arithmetic operators?

Reply:
Originally wanted these in the API for future rhs field expressions. Discussed offline -- we can add these back if we ever get around to that.